[Bug] [alert] alert http-plugin `Body` is not necessary but if not input any thing  got error log

### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar issues.


### What happened


I'm try to config a alert using `http` 

the config page show the `Body` is not necessary, but if not input any thing , got the error log, and not send the alert message.

![image](https://user-images.githubusercontent.com/25143332/228458564-1d493a21-8b1c-4efa-b0d4-4d0ce1ff6ccb.png)

```shell
[ERROR] 2023-03-29 07:19:32.245 +0000 org.apache.dolphinscheduler.plugin.alert.http.HttpSender:[164] - send http alert msg  exception : null
```



### What you expected to happen

Not input `Body` success.
or change the `Body` is must

### How to reproduce

don not input the  `Body` 

### Anything else

_No response_

### Version

dev

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

--- a/dolphinscheduler-alert/dolphinscheduler-alert-plugins/dolphinscheduler-alert-http/src/main/java/org/apache/dolphinscheduler/plugin/alert/http/HttpSender.java
+++ b/dolphinscheduler-alert/dolphinscheduler-alert-plugins/dolphinscheduler-alert-http/src/main/java/org/apache/dolphinscheduler/plugin/alert/http/HttpSender.java
@@ -163,7 +163,10 @@ public final class HttpSender {
      */
     private void setMsgInRequestBody(String msg) {
         try {
-            ObjectNode objectNode = JSONUtils.parseObject(bodyParams);
+            ObjectNode objectNode = JSONUtils.createObjectNode();
+            if (StringUtils.isNotBlank(bodyParams)) {
+                objectNode = JSONUtils.parseObject(bodyParams);
+            }
             // set msg content field
             objectNode.put(contentField, msg);
             StringEntity entity = new StringEntity(JSONUtils.toJsonString(objectNode), DEFAULT_CHARSET);

